solr/solr-ref-guide/src/index.adoc

```diff
@@ -121,6 +122,8 @@ The *<<getting-started.adoc#getting-started,Getting Started>>* section guides yo
 *<<solrcloud.adoc#solrcloud,SolrCloud>>*: This section describes SolrCloud, which provides comprehensive distributed capabilities.
 
 *<<legacy-scaling-and-distribution.adoc#legacy-scaling-and-distribution,Legacy Scaling and Distribution>>*: This section tells you how to grow a Solr distribution by dividing a large index into sections called shards, which are then distributed across multiple servers, or by replicating a single index across multiple services.
+
+*<<circuit-breakers.adoc#circuit-breakers,Circuit Breakers>>*: This section talks about circuit breakers, a comprehensive way of allowing a higher stability of Solr nodes and predictability of request execution.
```

Comment:
Erhm .. "a comprehensive way" it is not, but it is certainly "a way" :) I would also argue that it doesn't increase predictability of request execution because it introduces a new failure mode that clients have to handle - rather it's a way to ensure the service-level guarantees for requests that are accepted for execution.